### DISCUSSION TITLE
v2.9.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#5463] [Releases & Announcements]
## Changelog

- Help Center (Beta)
- Email conversation continuity for API channels
- Ability for Dashboard App Frames to query the information 
- Improved blocking & throttling of abusive requests
- Automations supporting case insensitive filters
- Numerous API enhancements and bug fixes

###
Thanks to @jordan-brough and @shivamsinghchahar  for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.9.0'>v2.9.0</a>.</em>

**Comment by glauberssoutodossantos** [comment]
Hi @sojan-official. How this new feature works?
`Ability for Dashboard App Frames to query the information`

**Comment by sojan-official** [comment]
## 2.9.0 Dev Changelog

### API Endpoints

#### Additions

- none

#### Deletions
- none

---

### Other Changes

- Helpcenter Endpoints
  - Description: Continuing the work over help center module, refactoring the routes as per product requirements
  - PR: https://github.com/chatwoot/chatwoot/pull/5309
